src/clj/rems/validate.clj

```diff
@@ -52,8 +52,10 @@
 
 (defn validate []
   (log/info "Validating configuration")
-  (when (:oidc-userid-attribute env)
-    (throw (ex-info "Please migrate to :oidc-userid-attributes" (select-keys env [:oidc-userid-attribute]))))
+  (when-some [oidc-attribute (:oidc-userid-attribute env)]
+    (throw (ex-info "Please migrate to :oidc-userid-attributes"
+                    {:from (select-keys env [:oidc-userid-attribute])
+                     :to {:oidc-userid-attributes [{:attribute oidc-attribute}]}})))
```

Comment:
I believe it could have been a sequence (as per https://github.com/CSCfi/rems/commit/1c83d3d1780bb8d6158e44561f8665ef4d1c3959) so maybe add a loop?

Reply:
ah thanks! i completely forgot and just looked at the single attribute as in the issue screenshot.